plotUtils/structFind6.py

```diff
@@ -0,0 +1,234 @@
+import ROOT as r
```

Comment:
I think this is in another PR, where I asked you to rename the file to be more descriptive of what the script does.

Reply:
I will remove this from this current PR